frontend/appflowy_flutter/lib/plugins/document/presentation/editor_plugins/header/document_cover_widget.dart

```diff
@@ -196,7 +196,7 @@ class _DocumentCoverWidgetState extends State<DocumentCoverWidget> {
               ],
             ),
             Padding(
-              padding: const EdgeInsets.only(bottom: 12.0),
```

Comment:
Should we add the right padding as well?

Reply:
you're right, I'll add the right padding